tests/cephfs/cephfs_pool_pg_size_change.py

```diff
@@ -0,0 +1,202 @@
+import datetime
+import logging
+import secrets
+import string
+import time
+import traceback
+
+from ceph.ceph import CommandFailed
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+logger = logging.getLogger(__name__)
+log = logger
+
+
+def check_clean_pgs(client, timeout=300):
+    """
+    Verify all pgs are active+clean
+    :param client: client node
+    :param timeout: timeout for checking pg state
+    """
+    timeout = datetime.timedelta(seconds=timeout)
+    starttime = datetime.datetime.now()
+    arg1 = "{print $1 }"
+    arg2 = "{print $3 }"
+    arg3 = "{print $4 }"
+    out, rc = client[0].exec_command(sudo=True, cmd=f"ceph pg stat | awk '{arg1}'")
```

Comment:
wtih the below way is this causing any issue 
out, rc = client[0].exec_command(sudo=True, cmd=f"ceph pg stat | awk '{{print $1 }}'").

we can avoid extra lines of code